model.py

```diff
@@ -11,33 +11,9 @@
 from torch.optim.lr_scheduler import StepLR
 from utils import datautils
 import joblib
-
-class referenceNetwork1(nn.Module):
-    '''
-    Network based on the description of the reference paper. Assumptions have been made to determine this
-    architecture since an exact description was not given. This Network is the best performing network on the
-    Drell-Yan Weights.
-    Reference:https://arxiv.org/pdf/2008.10949.pdf
-    '''
-    def __init__(self, n_particles):
-        super(referenceNetwork1, self).__init__()
-        self.fc1 = nn.Linear(n_particles, 200)
-        self.fc2 = nn.Linear(200, 200)
-        self.fc3 = nn.Linear(200, 200)
-        self.fc4 = nn.Linear(200, 200)
-        self.fc5 = nn.Linear(200, 200)
-        self.fc6 = nn.Linear(200, 1)
-        self.relu = nn.ReLU()
-        self.selu = nn.SELU()
-
-    def forward(self, x):
-        y = self.relu(self.fc1(x))
-        y = self.relu(self.fc2(y))
-        y = self.relu(self.fc3(y))
-        y = self.relu(self.fc4(y))
-        y = self.relu(self.fc5(y))
-        y = self.selu(self.fc6(y))
-        return y
+import hist
+from sklearn.metrics import mean_absolute_percentage_error as mape
+from networks import *
```

Comment:
In general we should always avoid doing `import *` as this basically totally destroys our namespace and makes it much more difficult to understand how changes in APIs propagate in the future. Can we explicitly give the imports here?